i18n/locales/de-DE.json

```diff
@@ -47,6 +47,10 @@
                     "label": "Wähle deine bevorzugte Sprache",
                     "info": "Um die Sprache der ganzen App zu ändern musst du GhostChat einmal neu starten."
                 },
+                "keybind-change": {
+                    "label": "Setze einen neue Vanish Tastenkombination",
+                    "info": "Control wird für STRG auf Windows/Linux verwendet, Command auf MacOS für die CMD Tate. Zum Beispiel: Alt+CommandOrControl+I wird Alt+CTRL+I auf Win/Linux und Alt+CMD+I auf MacOS. Weitere Kombinationen: CommandOrControl+I, Control+Alt+F9, Command+Alt+Shift+I."
```

Comment:
i think it would make sense to either add `\n` line breaks or move the info translation to an array and loop through it in the template